Marsiple/Common/Extensions/String-Marsiple.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  String-Marsiple.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 16/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+public extension String {
+    func localized(_ args: CVarArg...) -> String {
```

Comment:
Quick question - do you fully understand how this method works?

Reply:
Now that I've analyzed it, I fully understand how it works.